model/message/message.py

```diff
@@ -83,7 +83,7 @@ def populate(self, source, overwrite=True):
                     # Set the value of the variable on this object accessing it by name
                     setattr(self, key, value)
             else:
-                raise ValueError("Unexpected key encountered during Message population: '{key}'.")
+                raise ValueError(f"Unexpected key encountered during Message population: '{key}'.")
```

Comment:
very nice catch

Reply:
Only after spending 15 minutes sending messages and wondering why the key name is "key" 😄